pyramid_restful/api.py

```diff
@@ -124,31 +164,25 @@ def register(self, service, name=''):
         """
         # expose a sub-factory
         if isinstance(service, ApiFactory):
-            self.__services[name] = (service.factory, None)
```

Comment:
cleaned this all up...there are really only 3 use cases:

register an sub-api
register a module
register a class
register an endpoint

this was overcomplicated for what it was.  This in combination with the pyramid_orb updates don't break downstream components (tested against the tbx_api project)